content/operate/oss_and_stack/management/security/acl.md

```diff
@@ -274,17 +274,7 @@ really annoying, so instead we do things like this:
 
     > ACL SETUSER antirez on +@all -@dangerous >42a979... ~*
 
-By saying +@all and -@dangerous, we included all the commands and later removed
-all the commands that are tagged as dangerous inside the Redis command table.
-Note that command categories **never include modules commands** with
-the exception of +@all. If you say +@all, all the commands can be executed by
-the user, even future commands loaded via the modules system. However if you
-use the ACL rule +@read or any other, the modules commands are always
-excluded. This is very important because you should just trust the Redis
-internal command table. Modules may expose dangerous things and in
-the case of an ACL that is just additive, that is, in the form of `+@all -...`
```

Comment:
I belive we can leave this sentence as well. It explains ACL.

Reply:
Which of the three?

Reply:
Fixed in next push.